expression/builtin_time.go

```diff
@@ -2571,34 +2571,54 @@ func (c *makeTimeFunctionClass) getFunction(args []Expression, ctx context.Conte
 	if err := c.verifyArgs(args); err != nil {
 		return nil, errors.Trace(err)
 	}
-	sig := &builtinMakeTimeSig{newBaseBuiltinFunc(args, ctx)}
+	class, flen, decimal := args[2].GetType().ToClass(), 10, 0
```

Comment:
can we use `evalTp` instead of `TypeClass`, see #4394 for more detail

Reply:
OK. Fixed.